Position pos
float32[] laserX
float32[] laserY